include/h2o/http2_common.h

```diff
@@ -175,6 +175,11 @@ typedef struct st_h2o_http2_window_update_payload_t {
     uint32_t window_size_increment;
 } h2o_http2_window_update_payload_t;
 
+typedef struct st_h2o_settings_kvpair {
+    uint16_t key;
+    uint32_t value;
+} h2o_settings_kvpair_t;
```

Comment:
Maybe add "http2" to the type name? Use of uint16 for names and uint32 for values are specific to HTTP/2.

Reply:
Done.